assets/wizards/engagement/views/reader-activation/index.js

```diff
@@ -73,6 +74,18 @@ export default withWizardScreen( ( { wizardApiFetch } ) => {
 			.catch( setError )
 			.finally( () => setInFlight( false ) );
 	};
+	const resetEmail = postId => {
```

Comment:
Since this is a destructive action, I strongly suggest adding a confirmation (`confirmAction` from `newspack-components` - https://github.com/Automattic/newspack-plugin/blob/5ded1f95d27f9598b0df74ef60a0f4e9f64774a5/assets/components/src/utils/index.js#L16-L22)

Reply:
Great suggestion! Added confirmation in https://github.com/Automattic/newspack-plugin/pull/3049/commits/f33df76dc0ddef74a3d94f33c48c71d1c71fd1bb